Classes/LinphoneManager.m

```diff
@@ -1491,7 +1491,7 @@ - (void)createLinphoneCore {
 	libmswebrtc_init(f);
   
 	linphone_core_reload_ms_plugins(theLinphoneCore, NULL);
-    configH264HardwareAcell(true, true);
+    configH264HardwareAcell(true, false);
```

Comment:
Disable hardware accelerated decoder. This decoder has proven to be unstable during tests and can produce completely gray iframes.